### PR TITLE
APK_DEBUG_SYMBOL_PRESENT

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

**Review by PiranhaSa — APPROVED** [review]
lgtm!

File: MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/description.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The application should provide as little explanatory information as possible with the compiled code. Metadata such as debugging information, line numbers, and descriptive function or method names make the binary or byte-code easier to reverse engineer.
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/description.md:1]: This is still not telling the use what is the issue.

We should simply start with the application ships with debug symbols.

File: MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/description.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+It is noteworthy that most crash reporting tools support uploading symbols to perform stack trace symbolization and don't require symbols to be present in the application.
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/description.md:3]: We should add instructions on how to verify the symbols with command line tools.

File: MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/recommendation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Remove all symbols and debug data from the application. To do so, configure the build type to exclude debug information.
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/recommendation.md:1]: Missing how to remove them. There are command line tools to strip them, and compiler flags to remove them. etc.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**